src/applications/personalization/dashboard/components/FormItem.jsx

```diff
@@ -89,15 +84,7 @@ class FormItem extends React.Component {
       </DashboardAlert>
     );
 
-    const content = isExpired ? expiredView : activeView;
-    if (isAuthorizable) {
-      return (
-        <AuthorizationComponent formConfig={formConfig} isVisible>
```

Comment:
This was the only place the `AuthorizationComponent` was used and also the only reason all of the form configs were being loaded into the helper file in the first place. And the old, unused 686 was the only form that passed the `isAuthorizable()` test.